python/ray/rllib/test/test_checkpoint_restore.py

```diff
@@ -23,7 +23,8 @@ def get_mean_action(alg, obs):
     "ES": {
         "episodes_per_batch": 10,
         "train_batch_size": 100,
-        "num_workers": 2
+        "num_workers": 2,
+        "observation_filter": "MeanStdFilter"
```

Comment:
Can we also add ARS to this file?

Reply:
Weird, thought I added it. Adding it now.

Reply:
I think ARS is added below?